cluster/gpu-player.yaml

```diff
@@ -3,29 +3,21 @@ kind: Job
 metadata:
   name: minigo-gpu-player
 spec:
-  parallelism: $NUM_K8S_NODES
+  parallelism: 10
```

Comment:
Should this be configurable?

Reply:
nah.  it's not worth the math hassle, plus it's easier to fly it manually.